emeis/core/models.py

```diff
@@ -44,6 +44,7 @@ class BaseModel(VisibilityMixin, models.Model):
     created_at = models.DateTimeField(auto_now_add=True, db_index=True)
     modified_at = models.DateTimeField(auto_now=True, db_index=True)
     created_by_user = models.ForeignKey("User", null=True, on_delete=models.SET_NULL)
+    meta = JSONField(_("meta"), default=dict)
```

Comment:
The encoder should be a setting, not a translation string.

Reply:
This is not the encoder, It's the `verbose_name`.